test/system/250-systemd.bats

```diff
@@ -201,4 +202,62 @@ LISTEN_FDNAMES=listen_fdnames" "LISTEN Environment passed: $context"
     check_listen_env "$stdenv" "podman start"
 }
 
+@test "podman generate - systemd template" {
+    cname=$(random_string)
+    run_podman run -dt --name $cname $IMAGE top
```

Comment:
Can you use `create` instead of `run`?  That would consume fewer cycles.

Reply:
DONE :heavy_check_mark: